R-package/README.md

```diff
@@ -20,14 +20,52 @@ Note: 32-bit (i386) R/Rtools is currently not supported.
 
 #### Windows Preparation
 
-Installing [Rtools](https://cran.r-project.org/bin/windows/Rtools/) is mandatory, and only support the 64-bit version. It requires to add to PATH the Rtools MinGW64 folder, if it was not done automatically during installation.
+Installing a 64-bit version of [Rtools](https://cran.r-project.org/bin/windows/Rtools/) is mandatory.
 
-The default compiler is Visual Studio (or [VS Build Tools](https://visualstudio.microsoft.com/downloads/)) in Windows, with an automatic fallback to Rtools or any [MinGW64](https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/) (x86_64-posix-seh) available (this means if you have only Rtools and CMake, it will compile fine).
+After installing `Rtools` and `CMake`, be sure the following paths are added to the environment variable `PATH`. These may have been automatically added when installing other software.
 
-To force the usage of Rtools / MinGW, you can set `use_mingw` to `TRUE` in `R-package/src/install.libs.R`.
+* `Rtools` MinGW bin:
```

Comment:
Can simplify the Rtools example using two examples (to also cover the underscore case) such as:

`Rtools`:
- For `Rtools` 3.x, example: `C:\Rtools\mingw_64\bin`
- For `Rtools` 4.0, example: both `C:\rtools40\mingw64\bin` and `C:\rtools40\usr\bin`

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/3065/commits/d1d8022cc36b5bff616192474a938861fcb7afce